src/test/java/uk/gov/pay/products/resources/ProductsResourceTest.java

```diff
@@ -122,7 +122,7 @@ public void shouldSuccess_whenSavingAValidProduct_withAllFields() throws Excepti
 
     @Test
     public void shouldError_whenSavingAProduct_withMandatoryFieldsMissing() throws Exception {
-        givenSetup()
+        givenAuthenticatedSetup()
```

Comment:
I would also add a test to ProductResourceTest.java where no token is presented and should expect a 401.

Reply:
There is already a test doing that 
https://github.com/alphagov/pay-products/pull/10/files#diff-adde91191a7a84bbc8b3967f3b3fd8deR14 
However, I also feel its better to have a test for each resource add specifically for 401

Reply:
Addressed 👍